keyboards/id67/config.h

```diff
@@ -43,25 +44,53 @@
 #define MATRIX_COL_PINS { C7, F6, F5, F4, F1, B7, D5, D1, D2, D3, D4, D0, D6, D7, B4 }
 
 #define DIODE_DIRECTION COL2ROW
-
 #define RGB_DI_PIN F0
-#ifdef RGB_DI_PIN
-    #define RGBLED_NUM 77
-    #define RGBLIGHT_HUE_STEP 8
-    #define RGBLIGHT_SAT_STEP 8
-    #define RGBLIGHT_VAL_STEP 8
-    #define RGBLIGHT_LIMIT_VAL 150 /* The maximum brightness level */
-    #define RGBLIGHT_SLEEP  /* If defined, the RGB lighting will be switched off when the host goes to sleep */
-/*== all animations enable ==*/
-    #define RGBLIGHT_ANIMATIONS
-// /*== or choose animations ==*/
-//     #define RGBLIGHT_EFFECT_BREATHING
-//     #define RGBLIGHT_EFFECT_RAINBOW_MOOD
-//     #define RGBLIGHT_EFFECT_RAINBOW_SWIRL
-//     #define RGBLIGHT_EFFECT_SNAKE
-//     #define RGBLIGHT_EFFECT_KNIGHT
-//     #define RGBLIGHT_EFFECT_CHRISTMAS
-//     #define RGBLIGHT_EFFECT_STATIC_GRADIENT
-//     #define RGBLIGHT_EFFECT_RGB_TEST
-//     #define RGBLIGHT_EFFECT_ALTERNATING
-#endif
+#define DRIVER_LED_TOTAL 77
+#define DRIVER_LED_UNDERGLOW 10
+
+#if defined(RGB_DI_PIN)
```

Comment:
This `#if` not very useful (`RGB_DI_PIN` is defined just a couple of lines above).

However, you may consider replacing it with `#ifdef RGB_MATRIX_ENABLE`, and then restoring the rgblight config above wrapped in `#ifdef RGBLIGHT_ENABLE`; also wrap the `g_led_config` definition inside `#ifdef RGB_MATRIX_ENABLE`. Then someone who does not care about RGB Matrix could do this in their `rules.mk`:
```makefile
RGB_MATRIX_ENABLE = no
RGBLIGHT_ENABLE = yes
```
(this may be useful if some minimal lighting support is desired, but RGB Matrix does not fit into flash together with some other huge feature).

Reply:
Did exactly what you proposed, thanks!